mne/viz/_brain/tests/test.ipynb

```diff
@@ -39,16 +42,40 @@
     "initial_time = 0.13\n",
     "mne.viz.set_3d_backend('notebook')\n",
     "brain_class = mne.viz.get_brain_class()\n",
-    "for interactive_state in (False, True):\n",
-    "    plt.interactive(interactive_state)\n",
+    "\n",
+    "\n",
+    "@contextmanager\n",
+    "def interactive(on):\n",
+    "    old = plt.isinteractive()\n",
+    "    plt.interactive(on)\n",
+    "    try:\n",
+    "        yield\n",
+    "    finally:\n",
+    "        plt.interactive(old)\n",
+    "\n",
+    "with interactive(False):\n",
     "    brain = stc.plot(subjects_dir=subjects_dir, initial_time=initial_time,\n",
     "                     clim=dict(kind='value', pos_lims=[3, 6, 9]),\n",
     "                     time_viewer=True,\n",
-    "                     hemi='split')\n",
+    "                     show_traces=True,\n",
+    "                     hemi='split', size=300)\n",
     "    assert isinstance(brain, brain_class)\n",
     "    assert brain.notebook\n",
     "    assert brain._renderer.figure.display is not None\n",
     "    brain._update()\n",
+    "    total_number_of_buttons = len([k for k in brain.actions.keys() if '_field' not in k])\n",
+    "    number_of_buttons = 0\n",
+    "    for action in brain.actions.values():\n",
+    "        if isinstance(action, Button):\n",
+    "            action.click()\n",
+    "            number_of_buttons += 1\n",
+    "    assert number_of_buttons == total_number_of_buttons\n",
+    "    img_nv = brain.screenshot()\n",
+    "    assert img_nv.shape == (300, 300, 3), img_nv.shape\n",
```

Comment:
Notebook always renders non-HiDPI. At some point we should look into how to enable HiDPI support in the notebook -- basically we need the background renderer to be `ratio` (usually 2) times the size, and we need to tell it to display the image in native resolution (rather than treating physical and logical pixels the same). I suspect this is supported at some level in browsers, but TBD whether ipyvtk_simple allows setting this display resolution factor (i.e., to make images that are 600x600 pixels show up in a 300x300 logical pixel HTML element). Could be a cool PR there. Then we'd need a way to probe the physical-to-logical pixel ratio. Ideally this would come from the notebook but if not, we can quickly import Qt and query the screens and take a best guess...